app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsRoot.kt

```diff
@@ -378,7 +376,7 @@ fun AgendaDetailsContent(
                         )
                     }
                 }
-                item { Spacer(modifier = Modifier.weight(1f)) }
+                item { Spacer(modifier = Modifier.padding(100.dp)) }
```

Comment:
weight does not work inside of lazyColumn :(

Reply:
Yes, since a lazy column doesn't know all of its items (some of these might not be loaded due to the lazy behavior), it also can't calculate the dimensions of them all. Simple padding will be fine here. If you'd really like to have the weight behavior, you could use the lazy list state to check if it's scrollable and if not, calculate the remaining height of the screen after subtracting all visible elements to assign that for the spacer.